libraries/chain/include/scorum/chain/schema/comment_objects.hpp

```diff
@@ -128,6 +117,29 @@ class comment_vote_object : public object<comment_vote_object_type, comment_vote
     int8_t num_changes = 0;
 };
 
+class comment_statistic_object : public object<comment_statistic_object_type, comment_statistic_object>
```

Comment:
Try to use template with fields
asset total_payout;
asset author_payout;
etc...